pages/press.md

```diff
@@ -12,6 +12,15 @@ Are you looking to get a speaker or interview about Solid? Contact us on [press@
 
 ### 2021
 
+{%
+include talk.html
+  date="2021-04-01"
+  title="Solid World April"
+  website="https://vimeo.com/solidworld/review/531907097/eda903e37f"
```

Comment:
Since this is pointing to the same place as the video link, I think we might as well leave it out :)

```suggestion
```